int8 ID
int8 Power
---
int8 Success
